ui-v2/blueprints/adapter-test/index.js

```diff
@@ -0,0 +1,43 @@
+/*eslint node/no-extraneous-require: "off"*/
+'use strict';
+
+const useTestFrameworkDetector = require('@ember-data/-build-infra/src/utilities/test-framework-detector');
+const isModuleUnificationProject = require('@ember-data/-build-infra/src/utilities/module-unification')
```

Comment:
I think module unification is no longer happening so you could remove this and that should clear out a whole branch below too.